internal/python/plan_test.go

```diff
@@ -276,8 +276,8 @@ func TestHasDependency_Unknown(t *testing.T) {
 	}
 
 	// should always False
-	assert.False(t, HasDependency(ctx, "foo"))
-	assert.False(t, HasDependency(ctx, "bar"))
```

Comment:
Don't modify the current test. You can copy a new function based on your new `HasDependencyWithFile` with `TestHashDependencyWithFile_` prefixed.